software-requirement-specification.md

```diff
@@ -99,29 +106,37 @@ This section describes specific features of the software project. Specify the re
 
 ### 3.2 Non-Functional Requirements
 
-Non-functional requirements may exist for the following attributes. Often these requirements must be achieved at a system-wide level rather than at a unit level. State the requirements in the following sections in measurable terms (e.g., 95% of transactions shall be processed in less than a second, system downtime may not exceed 1 minute per day, > 30-day MTBF value, etc).
-
-This section describes non-functional features of the software project. Specify the requirements as user stories.
+This section outlines the non-functional requirements crucial for the U↑ internet-based phone system, focusing on performance, reliability, availability, security, and interfaces.
 
 #### 3.2.1 Performance
 
-- The product shall take initial load time depending on internet connection strength which also depends on the media from which the product is run.
+- **System Latency:** Call rerouting should happen within a 5-second window.
+- **Concurrency:** The system must support at least 500 concurrent calls without degradation in performance.
+- **Scalability:** The system should accommodate a 20% increase in user devices annually without requiring a system overhaul.
```

Comment:
might be worth adding "- The performance shall depend upon hardware components of the client/customer."

Reply:
I agree that this should go in the doc, but in my opinion it's not a non-functional requirement

is there a way I could rephrase it to make it a requirement, or is there a better place in the doc for this?